### ISSUE TITLE
Design System Staging Review - Accessibility Feedback - Minimal Footer

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#71936]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** The minimal footer component doesn't include a `footer` element or anything with the role `contentinfo`. Since pages on VA.gov with the standard footer do have a `contentinfo` landmark, it's possible that a screen reader user navigating by landmarks might be confused when they land on a page that doesn't have one.
   
   I think this is **very** unlikely to actually prevent any users from completing a task. Like, I have zero concerns about this shipping as-is. But for the sake of consistency, if the standard footer has a `contentinfo` role, then the visually-corresponding minimal footer should also have that role.
